src/NUnitFramework/framework/Attributes/RangeAttribute.cs

```diff
@@ -23,15 +23,25 @@
 
 using System;
 using System.Collections;
+using System.Reflection;
+
+using NUnit.Compatibility;
+using NUnit.Framework.Interfaces;
 
 namespace NUnit.Framework
 {
     /// <summary>
     /// RangeAttribute is used to supply a range of values to an
     /// individual parameter of a parameterized test.
     /// </summary>
-    public class RangeAttribute : ValuesAttribute
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true)]
```

Comment:
Most similar attributes are marked `Inherited = false`. (It's true by default.) Should this be consistent - or have I missed a design decision here? 🙂

Reply:
Good catch! AFAIK all our data attributes use `Inherited = false`

Reply:
I'm not entirely clear about what `Inherited = false` means in combination with `AttributeTargets.Parameter`. Can you help me understand what that does and why it's used on the other attributes?